lib/components/Body.js

```diff
@@ -50,6 +56,15 @@ var Body = React.createClass({
     if (isDate(body)) {
       return $(Data, null, body.toString());
     }
+    if (isSet(body)) {
+      var contents = Array.from(body)
```

Comment:
`function.name` has worked in browsers for ages - but I've also replaced those checks with node modules now anyway.

`Array.from` is ES6, and not well supported (same as `Object.assign`), so I'd rather use the package.